tests/pytests/test_dawn_drivers.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+import ale
+from ale.drivers import dawn_drivers
+from ale.base import data_naif
+from ale.base import label_pds3
+
+from unittest.mock import PropertyMock, patch
+
+# 'Mock' the spice module where it is imported
+from conftest import SimpleSpice, get_mockkernels
+
+simplespice = SimpleSpice()
+
+data_naif.spice = simplespice
+dawn_drivers.spice = simplespice
+label_pds3.spice = simplespice
+
+from ale.drivers.dawn_drivers import DawnFcPds3NaifSpiceDriver
+
+DawnFcPds3NaifSpiceDriver.metakernel = get_mockkernels
+
+@pytest.fixture
+def driver():
+    return DawnFcPds3NaifSpiceDriver("")
+
+def test_instrument_id(driver):
+    with patch('ale.base.label_pds3.Pds3Label.instrument_id', new_callable=PropertyMock) as mock_instrument_id:
+        mock_instrument_id.return_value = 1
+        with patch('ale.base.label_pds3.Pds3Label.filter_number', new_callable=PropertyMock) as mock_filter_number:
+            mock_filter_number.return_value = 2
+            assert driver.instrument_id == 'DAWN_1_FILTER_2'
+
+def test_spacecraft_name(driver):
+    with patch('ale.base.label_pds3.Pds3Label.instrument_host_id', new_callable=PropertyMock) as mock_host_id:
+        mock_host_id.return_value = 'DAWN'
+        assert driver.spacecraft_name == 'DAWN'
```

Comment:
This works with single patches as well and keeps things a little cleaner:
```
@patch('ale.base.label_pds3.Pds3Label.instrument_host_id', 'DAWN')
def test_spacecraft_name(driver):
    assert driver.spacecraft_name == 'DAWN'
```

Reply:
My one comment here is that it might be a bit more readable as: `@patch('ale.base.label_pds3.Pds3Label.instrument_host_id', return_value='DAWN')`

Just because we know that the second arg is actually a kwarg. - unless I am misreading the docs and this fails. This is really minor.

Reply:
Okay, everything has been changed to this new format. @jlaura it doesn't seem like the return_value thing works. I will update the wiki page to include this later today